src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -0,0 +1,142 @@
+            </muxc:NavigationViewItem>
+    
+            <muxc:NavigationViewItem x:Uid="Nav_Interaction"
+                                     Tag="Interaction_Nav">
+                <muxc:NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE7C9;"/>
+                </muxc:NavigationViewItem.Icon>
+            </muxc:NavigationViewItem>
+    
+            <muxc:NavigationViewItem x:Uid="Nav_Appearance"
+                                     Tag="GlobalAppearance_Nav">
+                <muxc:NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE771;"/>
+                </muxc:NavigationViewItem.Icon>
+            </muxc:NavigationViewItem>
+    
+            <muxc:NavigationViewItem x:Uid="Nav_ColorSchemes"
+                                     Tag="ColorSchemes_Nav">
+                <muxc:NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE790;"/>
+                </muxc:NavigationViewItem.Icon>
+            </muxc:NavigationViewItem>
+    
+            <muxc:NavigationViewItem x:Uid="Nav_Rendering"
+                                     Tag="Rendering_Nav" >
+                <muxc:NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE7F8;"/>
+                </muxc:NavigationViewItem.Icon>
+            </muxc:NavigationViewItem>
+    
+            <muxc:NavigationViewItemHeader x:Uid="Nav_Profiles"/>
+    
+            <muxc:NavigationViewItem x:Uid="Nav_ProfileDefaults"
+                                     Tag="GlobalProfile_Nav">
+                <muxc:NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE81E;"/>
+                </muxc:NavigationViewItem.Icon>
+            </muxc:NavigationViewItem>
+    
+        </muxc:NavigationView.MenuItems>
+        
+        <muxc:NavigationView.PaneFooter>
+            <!--The OpenJson item needs both Tapped and KeyDown handler-->
+            <muxc:NavigationViewItem x:Uid="Nav_OpenJSON"
+                                     Tapped="OpenJsonTapped"
+                                     KeyDown="OpenJsonKeyDown">
+                <muxc:NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE713;"/>
+                </muxc:NavigationViewItem.Icon>
+            </muxc:NavigationViewItem>
+        </muxc:NavigationView.PaneFooter>
+        
+        <Grid>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="*"></RowDefinition>
+                <RowDefinition Height="Auto"></RowDefinition>
+            </Grid.RowDefinitions>
+            <Frame x:Name="contentFrame"
+                   Grid.Row="0"></Frame>
+            <Grid Height="100" Grid.Row="1"
+                  BorderBrush="{ThemeResource SystemBaseLowColor}"
+                  BorderThickness="0,1,0,0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*"></ColumnDefinition>
+                </Grid.ColumnDefinitions>
+                <TextBlock x:Uid="Settings_UnsavedSettingsWarning"
```

Comment:
This currently isnt used, right?

Reply:
Correct. We need a way to detect unsaved changes to properly show/hide it. #8552 starts getting us there, so I'll keep it in there for now. But I'll definitely add a comment explaining that (and filing a follow-up)